src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,17 @@
 import React from 'react';
 import './MoviesList.scss';
+import { Movie } from '../../types/Movie';
+// eslint-disable-next-line import/no-cycle
+import { MovieCard } from '../MovieCard';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  dataMovie: Movie[];
+};
+
+export const MoviesList: React.FC<Props> = ({ dataMovie }) => (
+  <div className="movies">
+    {dataMovie.map((movie) => (
+      <MovieCard dataMovie={movie} key={movie.imdbId} />
```

Comment:
```suggestion
      <MovieCard movie={movie} key={movie.imdbId} />
```
![image](https://github.com/mate-academy/react_movies-list/assets/96409563/3e6f4d0a-5517-4de8-b436-4107079ee450)